### DISCUSSION TITLE
Installazione Home assistant core su docker compose non riuscita

### DISCUSSION THREAD

**Discussion by ghost** [indomus/forum#1298] [HUB: Home Assistant]
Salve, ho seguito la guida piu di una volta, ma il risultato e sempre lo stesso, home assistant core non si avvia, o se si avvia da una serie di errori, volevo ripartire da 0 e quindi rimuovere anche l'installazione di Docker e Docker compose, quali comandi devo eseguire per fare una pulizia completa? grazie

**Comment by snakuzzo** [comment]
Ma perché invece di ripartire da zero non ci fai vedere che errori hai? Se tra un anno hai un errore che fai? Riparti da zero?

**Comment by ghost** [comment]
Altro problema stavo seguendo la guida https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-installato-su-raspberry-raspbian/ 
per effettuare accesso da esterno ma quando provo ad accedere con utente homeassistant con il seguente comando :  sudo su -s /bin/bash homeassistant, come riporta la guida ricevo questo errore 
![Immagine](https://user-images.githubusercontent.com/27926995/149624824-501588ff-2a44-42a1-bf10-62e662f537f9.jpg)

**Reply by snakuzzo** [reply]
Perdonami ma Hai installato Core su Docker ... che c'entra quel comando?
Stai mischiando l'installazione fatta con Docker con quella senza...stai facendo un mezzo casino :)

Ti do un consiglio...se hai scelto di utilizzare Docker, lascia perdere dehydrated e segui la guida per Caddy su Docker (o docker-compose)

**Comment by vinny74man** [comment]
Ciao scusate sto da poco iniziando ad usare HA core con docker e volevo chiedere una cosa, poichè devo memorizzare i record di HA su mysql c'è la possibilità di installarlo nello stesso container di HA oppure no.

Grazie

**Reply by snakuzzo** [reply]
Quello che chiedi non ha alcun senso logico e vai proprio in contrasto con le logiche dei container.
Soluzione corretta...nel compose definisci anche il servizio mysql/mariadb e nella definizione del servizio di Core utilizzi `depends_on` specificando il servizio del DB

**Reply by vinny74man** [reply]
Scusa potresti indicarmi un esempio per capire come scrivere il docker-compose file per fare questo che mi hai suggerito?
Grazie

**Reply by snakuzzo** [reply]
[QUI](https://docs.docker.com/compose/) (ma anche [QUI](https://mariadb.com/kb/en/setting-up-a-lamp-stack-with-docker-compose/)) trovi tutto quello che ti serve 
Ogni container un'applicazione e li fai comunicare tra loro in base alle esigenze.

**Reply by indomus** [reply]
e [QUI](https://indomus.it/formazione/docker-compose-cose-a-cosa-serve-e-perche-si-usa/).
